content/home/marketing/declarative.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Declarative
+title: הצהרתי
 order: 0
 ---
 
-React makes it painless to create interactive UIs. Design simple views for each state in your application, and React will efficiently update and render just the right components when your data changes.
+React הופך את היצירה של ממשקי משתמש לנטולת כאבים. עצב תצוגה פשוטה לכל state בישום שלך וReact יעדכן ויעבד בצורה אפקטיבית רק את הקומפוננטות הנכונות כאשר המידע שלך משתנה.
```

Comment:
Instead of using לנטולת כאבים maybe use something else like ליותר נוחה?

Reply:
fixed it, although when you say `יותר נוחה` it's usually as a comparison to something else - and we don't have any comparison here

Reply:
Yeah you're right, it just reads better than לנטולת כאבים